src/app/shared/entry.ts

```diff
@@ -461,6 +461,16 @@ export abstract class Entry implements OnDestroy {
    * @param topicId The ID of the topic on discourse
    */
   discourseHelper(topicId: number): void {
+    // If the #discourse-comments element does not exist, or if the previous topicId
```

Comment:
> If the #discourse-comments element 

Suggest something like "If the #discourse-comments placeholder element..." -- I had to go look at the code to understand this, and was initially confused why we you are checking for existence of an element I assumed that the Discourse JavaScript creates. And then I realized it's defined in our HTML.

Reply:
I added a sentence to the front of this comment that explains the purpose and provenance of the #discourse-comments element.